sechub-pds-solutions/owasp-zap/docker/scripts/owasp-zap.sh

```diff
@@ -18,7 +18,7 @@ owasp-zap -daemon -silent -nostdout -host "$ZAP_HOST" -port "$ZAP_PORT" -config
 echo "Waiting for OWASP-ZAP to start"
 TRIES=20
 WAITRETRY=6
-TOTAL_WAITTIME=$( echo "$TRIES*$WAITRETRY" | bc )
+TOTAL_WAITTIME=$(($TRIES*$WAITRETRY))
```

Comment:
I think it is easier to read if there is an underscore:

```suggestion
TOTAL_WAITTIME=$(($TRIES*$WAIT_RETRY))
```